src/index.js

```diff
@@ -300,39 +335,32 @@ module.exports = (
             );
 
             if (cleanBody !== undefined) {
-              Object.keys(cleanBody).forEach(name => {
-                // We neither have an easy way to transform `name` into `name[]` to signify that it's an array payload
-                // (and for all we know it might be an array of objects!), but also at the same time the HAR spec
-                // doesn't give any guidance for these kinds of cases so instead we're just falling back to
-                // stringifying the content instead of potentially including `fileName` properties.
-                if (Array.isArray(cleanBody[name])) {
-                  har.postData.params.push({
-                    name,
-                    value: JSON.stringify(cleanBody[name]),
-                  });
+              const multipartParams = multipartBodyToFormatterParams(
+                formData.body,
```

Comment:
This work might break on a `multipart/form-data` request that isn't an object, like this:

![Screen Shot 2022-01-24 at 11 30 31 AM](https://user-images.githubusercontent.com/33762/150851103-30919f88-5b6e-426c-82e5-ab73d1da6ce8.png)

`formData.body` here is just a string.